app/models/message_model.py

```diff
@@ -19,4 +18,4 @@ class MessageModel(db.Model):
     text = Column(String, nullable=False)
     date = Column(DateTime, nullable=False)
 
-    users = db.relationship("UserModel", secondary=users_messages, backref="messages")
+    # users = db.relationship("UserModel", secondary=users_messages, backref="messages")
```

Comment:
You can delete that, i ran into the same error, the backref property already created the "messages" in the other table, i might be wrong, but if we wanna keep that in both tables, we can use back_populates instead.

Reply:
I already fix it in my last pull request. You can delete it.